src/containers/Instagram/IgConnect.js

```diff
@@ -0,0 +1,241 @@
+import React, { useEffect, useState } from 'react'
+import { images } from '../../assets/images'
+import { connect, useSelector } from 'react-redux'
+import {
+  fetchUserMedia,
+  CrawlNewUser,
+  setIgConnectSearchText,
+  setSelectedMedia
+} from '../../actions/DataActions'
+import { Link } from 'react-router-dom'
+import { pageSize } from '../../config/static'
+import Pagination from 'react-js-pagination'
+import { ToastsStore } from 'react-toasts'
+import * as moment from 'moment-timezone'
```

Comment:
Can we avoid using moment package, if so then great, if not then try to find the reduce the footprint of moment package, as it brings on all the locales by default thus leaving a big footprint on the app.

Reply:
Also, you may not want to `import * as moment` since it brings in the complete moment library, our intention is so to precisely bring in what is actually needed.